src/apis/DataDisplayApi.ts

```diff
@@ -0,0 +1,18 @@
+
+import axios from 'axios';
+import { DataDisplayRequest, DataDisplayResponse } from '../types/Types';
+
+const BASE_URL = 'YOUR_API_BASE_URL';
+
+const DataDisplayApi = {
+  getDataDisplay: async (request: DataDisplayRequest): Promise<DataDisplayResponse> => {
+    try {
+      const response = await axios.get(`${BASE_URL}/data-display`);
```

Comment:
This is the complete url for data display:
https://0fg9aos700.execute-api.ap-south-1.amazonaws.com/asdfasdf/sample-route-key

Please use this